modular_skyrat/modules/modular_items/code/psych_cart.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/cartridge/psychologist
+	name = "\improper P.A.T cartridge"
+	desc = "Flaunt that Psychology Degree with this Cartridge!"
+	icon_state = "cart-m"
+	access = CART_MEDICAL
+	spam_enabled = 1
```

Comment:
Gotta use dem defines boi
```suggestion
	spam_enabled = TRUE
```

Reply:
Ack I forgot that simple rule, not making PRs for so long made me forget that